dependencies.go

```
@@ -35,12 +32,8 @@ func (f FastlaneRunner) InstallDependencies(opts EnsureDependenciesOpts) error {
 			f.logger.Println()
 
 			if err := cmd.Run(); err != nil {
-				var exitErr *exec.ExitError
-				if errors.As(err, &exitErr) {
-					return fmt.Errorf("command failed with exit status %d (%s)", exitErr.ExitCode(), cmd.PrintableCommandArgs())
-				}
-
-				return fmt.Errorf("executing command failed (%s): %w", cmd.PrintableCommandArgs(), err)
+				errorString := f.formattedCommandErrorMessage(cmd, err)
+				f.logger.Errorf(errorString)
```

here we should return an error and leave the error handling on the consumer of this function